### ISSUE TITLE
Can't install sorbet.

### ISSUE THREAD

**Issue by Starlexxx** [sorbet/sorbet#7182]
Hey, I'm using:
* MacOS Ventura 13.3.1
* ruby 2.7.5
* rvm
I've added this to my Gemfile:
```ruby
gem 'sorbet-runtime', '0.5.10932'
gem 'sorbet', '0.5.10932', :group => :development
```
After running ```bundle``` I get:
```shell
Fetching gem metadata from http://rubygems.org/........
Resolving dependencies...................
Could not find gem 'sorbet-static (= 0.5.10932)' with platform 'ruby', which is required by gem 'sorbet (= 0.5.10932)', in rubygems repository http://rubygems.org/ or installed locally.

The source contains the following gems matching 'sorbet-static (= 0.5.10932)':
  * sorbet-static-0.5.10932-java
  * sorbet-static-0.5.10932-universal-darwin-14
  * sorbet-static-0.5.10932-universal-darwin-15
  * sorbet-static-0.5.10932-universal-darwin-16
  * sorbet-static-0.5.10932-universal-darwin-17
  * sorbet-static-0.5.10932-universal-darwin-18
  * sorbet-static-0.5.10932-universal-darwin-19
  * sorbet-static-0.5.10932-universal-darwin-20
  * sorbet-static-0.5.10932-universal-darwin-21
  * sorbet-static-0.5.10932-universal-darwin-22
  * sorbet-static-0.5.10932-x86_64-linux

Process finished with exit code 6
```
Can you please help me?

**Comment by stephenbinns** [comment]
sorbet-static doesn't provide a gem for the `ruby` platform -  you may also need to add the plaform to the Gemfile

e.g. for OSX
```
bundle lock --add-platform universal-darwin-22
```
You'll have to do similar for any other platforms you use (e.g. CI)

**Comment by jez** [comment]
Should be fixed in #7292
